train_dolly.py

```diff
@@ -137,18 +135,40 @@
 local_output_dir = os.path.join(local_training_root, checkpoint_dir_name)
 dbfs_output_dir = os.path.join(dbfs_output_root, checkpoint_dir_name)
 
-num_gpus_flag = ""
-num_gpus = dbutils.widgets.get("num_gpus")
-if num_gpus:
-    num_gpus = int(num_gpus)
-    num_gpus_flag = f"--num_gpus={num_gpus}"
+# pick an appropriate config file
+gpu_family = dbutils.widgets.get("gpu_family")
+root_path = os.getcwd()
+deepspeed_config = os.path.join(root_path, f"config/{gpu_family}_config.json")
 
 tensorboard_display_dir = f"{local_output_dir}/runs"
 
 print(f"Local Output Dir: {local_output_dir}")
 print(f"DBFS Output Dir: {dbfs_output_dir}")
 print(f"Tensorboard Display Dir: {tensorboard_display_dir}")
 
+# configure the batch_size
+batch_size = 3
+if gpu_family == "a100":
+    batch_size = 6
+
+# configure CUDA memory management
+if gpu_family == "v100":
+    os.environ["PYTORCH_CUDA_ALLOC_CONF"] = "max_split_size_mb:64"
```

Comment:
Does this do anything? I don't think it really would

Reply:
I was getting errors without this set. I can test it again without, but it seemed necessary.

Reply:
If you run out of mem, the default error message mentions this config, but, in my experience it has never ever been the actual issue.

Reply:
Hmmm - just tested it again, and seems to be working fine without this. I'll take it out for now.